src/components/10-atoms/progress-bar/index.js

```diff
@@ -0,0 +1,93 @@
+import { LitElement, html, css, unsafeCSS } from 'lit-element';
+import { classMap } from 'lit-html/directives/class-map';
+import { defineVersioned } from '../../../utils/component-versioning';
+import { applyDefaults } from '../../../utils/with-react';
+import styles from './index.scss';
+
+class AXAProgressBar extends LitElement {
+  static get tagName() {
+    return 'axa-progress-bar';
+  }
+
+  static get styles() {
+    return css`
+      ${unsafeCSS(styles)}
+    `;
+  }
+
+  static get properties() {
+    return {
+      small: { type: Boolean, defaultValue: false },
+      fullWidth: { type: Boolean, defaultValue: false },
```

Comment:
This is pascalCase and therefore correct!

But what you document are "attributes" and those are in kebab case. LitElement will take care of the conversion. So what I mean up there in the readme: You declare the api there and there is has to be `full-width`, because HTML does not care about uppercase.